cypress/integration/console/devices/create-manual.spec.js

```diff
@@ -44,36 +39,39 @@ describe.skip('End device manual create', () => {
 
     beforeEach(() => {
       cy.loginConsole({ user_id: user.ids.user_id, password: user.password })
-      cy.visit(`${Cypress.config('consoleRootPath')}/applications/${appId}/devices/add/manual`)
+      cy.visit(`${Cypress.config('consoleRootPath')}/applications/${appId}/devices/add`)
+      cy.findByLabelText('Enter end device specifics manually').check()
     })
 
-    it('validates before submitting an empty form', () => {
-      cy.findByText('Show advanced activation, LoRaWAN class and cluster settings').click()
-      cy.findByLabelText('Over the air activation (OTAA)').check()
-
-      cy.findByRole('button', { name: 'Register end device' }).click()
-      cy.location('pathname').should(
-        'eq',
-        `${Cypress.config('consoleRootPath')}/applications/${appId}/devices/add/manual`,
-      )
+    it('displays UI elements in place', () => {
       cy.findByTestId('full-error-view').should('not.exist')
       cy.findByTestId('error-notification').should('not.exist')
 
-      cy.findErrorByLabelText('LoRaWAN version')
-        .should('contain.text', 'LoRaWAN version is required')
-        .and('be.visible')
-      cy.findErrorByLabelText('Regional Parameters version')
-        .should('contain.text', 'Regional Parameters version is required')
-        .and('be.visible')
-      cy.findErrorByLabelText('Frequency plan')
-        .should('contain.text', 'Frequency plan is required')
-        .and('be.visible')
+      cy.findByLabelText('Frequency plan').should('be.visible')
+      cy.findByLabelText('LoRaWAN version').should('be.visible')
+      cy.findByLabelText('Regional Parameters version').should('be.visible')
+
+      cy.findByText(
+        'Please enter versions and frequency plan information above to continue',
+      ).should('be.visible')
+    })
+
+    it('validates empty form before submitting', () => {
```

Comment:
Let's add an additional test here that checks whether missing MAC Settings are also validated. So that would mean unchecking the `Network defaults` checkbox and removing a value from a required field, submitting, and then checking whether the validation message appears.